src/Hocon/HoconParser.cs

```diff
@@ -38,9 +38,9 @@ public sealed class HoconParser
         ///     This exception is thrown when an unresolved substitution is encountered.
         ///     It also occurs when any error is encountered while tokenizing or parsing the configuration string.
         /// </exception>
-        public static HoconRoot Parse(string text, HoconIncludeCallbackAsync includeCallback = null)
+        public static HoconObject Parse(string text, HoconIncludeCallbackAsync includeCallback = null)
```

Comment:
`HoconRoot` only used to transfer substitution from included hocon files, they are not publicly exposed anymore.
`Parse()` returns a `HoconObject` instance instead.